client/asset/zec/zec.go

```diff
@@ -320,7 +320,7 @@ func (w *zecWallet) CallRPC(method string, args []any, thing any) error {
 
 // FeeRate returns the asset standard fee rate for Zcash.
 func (w *zecWallet) FeeRate() uint64 {
-	return dexzec.LegacyFeeRate
+	return 5000 // per logical action
```

Comment:
Why is `dexzec.LegacyFeeRate` no longer suitable?

Reply:
That's not how fees work for Zcash anymore. See [ZIP-0317](https://zips.z.cash/zip-0317).